Detector/DetSegmentation/src/GridDriftChamber.cpp

```diff
@@ -9,46 +10,93 @@ GridDriftChamber::GridDriftChamber(const std::string& cellEncoding) : Segmentati
   _type = "GridDriftChamber";
   _description = "Drift chamber segmentation in the global coordinates";
 
+  registerParameter("cell_size", "cell size", m_cellSize, 0., SegmentationParameter::LengthUnit);
+  registerParameter("offset_phi", "offset in phi", m_offsetPhi, 0., SegmentationParameter::LengthUnit, true);
+  registerParameter("detector_length", "Length of the wire", m_detectorLength, 1., SegmentationParameter::LengthUnit);
   registerIdentifier("identifier_phi", "Cell ID identifier for phi", m_phiID, "phi");
-  registerParameter("delta_phi", "delta phi", m_delta_phi, 0., SegmentationParameter::LengthUnit);
 }
 
 GridDriftChamber::GridDriftChamber(const BitFieldCoder* decoder) : Segmentation(decoder) {
-  // define type and description
+
   _type = "GridDriftChamber";
   _description = "Drift chamber segmentation in the global coordinates";
 
+  registerParameter("cell_size", "cell size", m_cellSize, 1., SegmentationParameter::LengthUnit);
+  registerParameter("offset_phi", "offset in phi", m_offsetPhi, 0., SegmentationParameter::LengthUnit, true);
+  registerParameter("epsilon0", "epsilon", m_epsilon0, 0., SegmentationParameter::LengthUnit, true);
```

Comment:
The parameter epsilon should be in the angle unit. Could have use `SegmentationParameter::AngleUnit` instead of the `LengthUnit`?

Tao